### DISCUSSION TITLE
Option groups on autocomplete

### DISCUSSION THREAD

**Discussion by ninhorata** [formkit/formkit#1021] [General]
Hello!

Is there a way to have option groups like on the Select component but on the AutoComplete one? The select one doesn't allow any typing, and I would like to have that behavior activated.

Thank you!

**Comment by sashamilenkovic** [comment]
@ninhorata Hey there! As of FormKit Pro v0.123.0, option groups are available to the `autocomplete`, `dropdown`, and `taglist` inputs: https://formkit.com/inputs/autocomplete#option-groups
